server/controllers/auth.controller.js

```diff
@@ -200,6 +240,7 @@ function register(req, res, next) {
 
       const newUser = new User();
       newUser.password = User.generateHash(password);
+      newUser.signedupForNewsletter = newsletterSignup; // Probably works
```

Comment:
Yeah this should work. Since what we want is the intention and not the actual success status. It's just in case the mailchimp code did fail for some reason, we could reconstruct this list later on.